packages/edge-demo-site/app/entry.server.tsx

```diff
@@ -1,28 +1,27 @@
-import type { EntryContext } from '@remix-run/server-runtime'
+/**
+ * By default, Remix will handle generating the HTTP Response for you.
+ * You are free to delete this file if you'd like to, but if you ever want it revealed again, you can run `npx remix reveal` ✨
+ * For more information, see https://remix.run/file-conventions/entry.server
+ */
+
+import type { AppLoadContext, EntryContext } from '@remix-run/node'
 import { RemixServer } from '@remix-run/react'
-import { renderToReadableStream } from 'react-dom/server'
```

Comment:
@nickytonline It seems like the original did use `renderToReadableStream` though 🤔

Reply:
That's correct. I removed it due to some DX issues, but I will most likely be adding it back. The Remix dev server doesn't support custom servers, which is why I went with the non-streaming version that runs in a Node context. It applies less to this repo the DX issue. It's more in the Remix template repo. It's related to the discussion I linked.